auth-api/src/auth_api/utils/roles.py

```diff
@@ -26,6 +26,7 @@ class Role(Enum):
     ACCOUNT_HOLDER = 'account_holder'
     GOV_ACCOUNT_USER = 'gov_account_user'
     ANONYMOUS_USER = 'anonymous_user'
+    ACCOUNT_IDENTITY = 'account_identity'
```

Comment:
Any idea what this `account_identity` is? something new I'm guessing?

Reply:
yeah this is the role for being allowed to check if a business identifier and an account id are affiliated. Its an IDIM flow (they call legal api for business json and if they provide an account id we allow them to check if the id is affiliated with the business)